protoc-gen-pbandk/lib/src/commonMain/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -226,21 +218,19 @@ public open class CodeGenerator(
         chunk: List<Pair<File.Field.Numbered, File.Field.OneOf?>>,
         fullTypeName: String
     ) {
-        line("fieldsList.apply {").indented {
```

Comment:
This change only removed the `fieldsList.apply {` line (and accompanying indentation). The rest of the code didn't change.